src/serverMain/kotlin/com/studo/campusqr/database/Payloads.kt

```diff
@@ -76,6 +76,21 @@ class BackendAccess : MongoMainEntry(), ClientPayloadable<ClientAccessManagement
   override fun toClientClass(language: String) = throw NotImplementedError()
 }
 
+class BackendSeatFilter : MongoMainEntry(), ClientPayloadable<ClientSeatFilter> {
+  lateinit var locationId: String
+  lateinit var editedBy: String // userId
+  lateinit var lastEditDate: Date
+  var seat: Int = 0
```

Comment:
Why give it a default?

Reply:
kotlin limitation: primitives can't be lateinit